app/src/main/java/org/schabi/newpipe/util/ThemeHelper.java

```diff
@@ -288,4 +290,21 @@ public static boolean isDeviceDarkThemeEnabled(final Context context) {
                 return false;
         }
     }
+
+    public static void setDayNightMode(final Context context) {
+        setDayNightMode(context, ThemeHelper.getSelectedThemeKey(context));
+    }
+
+    public static void setDayNightMode(final Context context, final String selectedThemeKey) {
+        final Resources res = context.getResources();
+
+        if (selectedThemeKey.equals(res.getString(R.string.light_theme_key))) {
+            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
+        } else if (selectedThemeKey.equals(res.getString(R.string.dark_theme_key))
+                || selectedThemeKey.equals(res.getString(R.string.black_theme_key))) {
+            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
+        } else {
+            AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM);
+        }
+    }
 }
```

Comment:
Check if `resolveResourceIdFromAttr` can be removed, and remove it if that's the case

Reply:
It can be removed

Reply:
Remove it ;-)